draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -105,7 +105,7 @@ values not disclosed in the Disclosures.
 This document also defines a format for SD-JWTs with Key Binding (SD-JWT+KB).
 By optionally sending an SD-JWT+KB to a
 Verifier, the Holder can prove to the Verifier that they hold the private key
-associated to the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
+associated with the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
```

Comment:
I would ask that Mike's feedback not overwrite the phrasing that seemed more natural to Martin Thomson https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/394/files/d150ec24a8a369af4976c2739d8b97ac5650e603#r1416409939 

```suggestion
associated to the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
```

Reply:
Jeez, such an impactful change hidden away in this unconspicious PR... thanks, Brian, for catching that.

Reply:
I am very sorry I did not catch that. you are absolutely right, reverting this.